modules/lmdb/project.clj

```diff
@@ -11,6 +11,13 @@
 
   :scm {:dir "../.."}
 
+  :java-source-paths ["src"]
```

Comment:
you'll need to pull in the javadoc and sources classifiers from another module btw (e.g. core) - now that you've got a public Java class Maven Central will expect javadoc and sources jars